org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -135,8 +135,17 @@
             "sources": [
                 {
                     "type": "file",
-                    "url": "https://github.com/rust-analyzer/rust-analyzer/releases/download/2020-12-28/rust-analyzer-linux",
-                    "sha256": "6e390a789c997ec5f6eb4b536c2c9395f5374e8815ce7fc4a51dce41cd44cd19"
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/rust-analyzer/rust-analyzer/releases/download/2021-01-11/rust-analyzer-linux",
```

Comment:
Don't update it and let's see if the checker works :)

Reply:
I tested it on my PC with the docker thingy, but sure :)